apps/blockchain/scripts/sync_with_infura.ex

```diff
@@ -1,6 +1,7 @@
 require Logger
 
 defmodule SyncWithInfura do
+  @save_block_interval 1000
```

Comment:
Maybe we can add a couple of saving points for debugging purposes?

Reply:
Like save at certain blocks like `500000`? If we want to stop at a certain block we've been adding an if statement inside `add_block_to_tree`. Could we achieve the same goal by doing that?